### DISCUSSION TITLE
沉浸式翻译报错

### DISCUSSION THREAD

**Discussion by youhaozhao** [BurnHair/docs#185] [技术支持]
报错信息:
```txt
500: {"error":{"message":"The server had an error while processing your request. Sorry about that! (request id: 202408171614176096761825xLV5KI5)","type":"server_error","param":"","code":null}}
```
 API 接口地址：https://burn.hair

**Comment by BennyThink** [comment]
使用的是什么模型，是否有参考 https://github.com/BurnHair/docs/blob/master/lobe-chat%20%E5%92%8C%20ChatGPT-Next-Web%20%E4%BD%BF%E7%94%A8.md 进行配置？

**Reply by youhaozhao** [reply]
我用gpt-4o-mini会报这个错误，gpt-4o就没有这个问题，后者工作的很好。
我之前参考了链接中的内容进行配置，在此之前gpt-4o-mini也能正常翻译🤔

**Comment by meishimeiyi** [comment]
我也遇到了此问题。我在沉浸式翻译里粘贴了从 burn[.]hair 得到的 api key, 选择 gpt-4o-mini 和 gpt-4o-mini-2024-07-18 ，配置自定义 API 接口地址为 https://burn.hair/v1/chat/completions 点击测试服务后报错。
报错信息：503: {"error":{"message":"当前分组 vip 下对于模型 gpt-4o-mini 无可用渠道 (request id: 20240817164558421649995vasbl82M)","type":"one_api_error"}}

**Comment by BennyThink** [comment]
区域出问题了，已经切到其他区域了，现在应该好了
